tutorials/intro-to-geospatial-analysis/index.md

```diff
@@ -0,0 +1,449 @@
+```
+### Extract values of a property from a Feature
+```javascript
+var featVal = feature.get("size");
+```
+---
+
+## Filters
+
+### Creator a filter for values of a property
+
+```javascript
+var bFilter = ee.Filter.eq(propertyName, value);
+```
+>or .neq , .gt , .gte , .lt , and .lte
+
+### Create a filter based on maximum difference from a threshold
+
+```javascript
+var diffFilter = ee.Filter.maxDifference(threshold, propertyName, value);
+```
+
+### Create a text filter
+
+```javascript
+var txtFilter = ee.Filter.stringContains(propertyName, stringValue);
+```
+
+>or .stringStartsWith, and .stringEndsWith
+
+### Create a range filter
+
+```javascript
+var rangeFilter = ee.Filter.rangeContains(propertyName, stringValue, minValue, maxValue);
+```
+  
+### Create a list filter to check for certain values
+
+```javascript
+var listFilter = ee.Filter.listContains(propertyName, value1, propertyName2, value2);
+```
+>.inList to test against list of values
+
+### Create a filter of dates
+
+```javascript
+var dateFilter = ee.Filter.calendarRange(startDate, stopDate);
+```
+
+### Create a filter for particular days of the year
+
+```javascript
+var dayFilter = ee.Filter.dayOfYear(startDay, stopDay);
+```
+
+### Create a filter to subset geospatial data
+
+```javascript
+var boundsFilter = ee.Filter.bounds(geometryOrFeature);
+```
+### Combining and inversing filters
+
+```javascript
+var newFilter = ee.Filter.and(listOfFilters);
+var newFilter = ee.Filter.or(listOfFilters);
+var inverseFilter = ee.Filter.not(filter);
+```
+---
+## Operations on Images
+
+[Image operations](https://code.earthengine.google.com/234ba0ab11dd82112602de05c0bc3bd8 "EE Image operations")
+
+### Selecting the bands of an image
+```javascript
+var band = var_Image.select(bandName);
+```
+### Creating masks
+```javascript
+var mask = image.eq(value);
+```
+>or .neq or .gt or .gte or .lt or .lte
+
+### Applying masks
+```javascript
+var masked = image.mask(mask);
+```
+
+### Pixelwise calculation
+```javascript
+var results = image.sum(value);
+```
+>or .subtract ,    .multiply ,    .divide ,    .max , .min ,  .abs ,  .round ,  .floor ,  .ceil ,  .sqrt ,  .exp,  .log, .log10, .sin ,  .cos ,  .tan ,  .sinh ,  .cosh ,  .tanh ,  .acos, .asin
+
+### Shift pixels of an image
+```javascript
+newImage = oldImage.leftShift(valueOfShift);                   
+```
+>or .rightShift
+
+### Create a single value from an image by applying a reducer based on regions of interest
+
+```javascript
+var outputDictionary = var_Image.reduceRegion(reducer, geometry, scale);
+```
+
+---
+
+# Operations on Image Collections
+
+### Select the first n numbers of images in a collection (based on property)
+
+```javascript
+var selectedImages = imCollection.limit(n, propertyName, order);
+```
+
+### Select images in collection based on particular properties
+
+```javascript
+var selectedIm = imCollection.filterMetadata(propertyName, relation , value);
```

Comment:
Please use .filter() (not .filterMetadata)

Reply:
Like this?

var selectedIm = imCollection.filter(ee.Filter.metadate(propertyName, relation , value));

Reply:
I have added this as an alternative option.

Reply:
alternative preferable option